ext/configuration.h

```diff
@@ -171,6 +174,7 @@ enum ddtrace_sampling_rules_format {
     CONFIG(CUSTOM(STRING), DD_TRACE_CLIENT_IP_HEADER, "", .parser = ddtrace_parse_client_ip_header_config)     \
     CONFIG(BOOL, DD_TRACE_FORKED_PROCESS, "true")                                                              \
     CONFIG(INT, DD_TRACE_HOOK_LIMIT, "100")                                                                    \
+    CONFIG(INT, DD_TRACE_AGENT_FLUSH_SIZE, "1048576", .ini_change = zai_config_system_ini_change)              \
```

Comment:
In [.NET](https://github.com/search?type=code&q=org%3ADataDog+DD_TRACE_BUFFER_SIZE) we use `DD_TRACE_BUFFER_SIZE`. I'd recommend using the same for consistency